mesmerize_napari/main_offline_gui.py

```diff
@@ -339,18 +340,27 @@ def load_output(self):
     def load_correlation_image(self):
         s = self.selected_series()
         corr_img = s.caiman.get_correlation_image()
+        if s['algo'] == 'cnmfe':
+            pnr_img = s.caiman.get_pnr_image()
+            self.viewer.add_image(pnr_img, name=f'pnr: {s["name"]}', colormap='gnuplot2')
         self.viewer.add_image(corr_img, name=f'corr: {s["name"]}', colormap='gnuplot2')
 
     def view_projections(self):
         proj_type = self.ui.comboBoxProjectionOpts.currentText()
-        item_gui = QtWidgets.QListWidgetItem = self.ui.listWidgetItems.currentItem()
-        uuid = item_gui.data(3)
-        # Algorithm name for selected item
-        algo = self.dataframe.loc[self.dataframe['uuid'] == uuid, 'algo'].item()
-        r = self.dataframe.loc[self.dataframe['uuid'] == uuid]  # pandas Series corresponding to this item
-        getattr(algorithms, algo).load_projection(self.viewer, r, proj_type)
-
-
+        s = self.selected_series()
+        projection = s.caiman.get_projection(proj_type=proj_type)
+        self.viewer.add_image(projection, name=f'{proj_type} projection {s["name"]}', colormap='gnuplot2')
+
```

Comment:
I guess you'll add a `view_pnr_image()` method here? Coupled to a slider to change the vmin, vmax?